code/datums/abilities/flock/flockmind.dm

```diff
@@ -266,7 +264,7 @@
 	for(var/obj/machinery/door/airlock/A in range(10, holder.owner))
 		if(A.canAIControl())
 			targets += A
-	if(targets.len > 1)
+	if(length(targets))
```

Comment:
those are not equivalent

Reply:
I see they're not now, but maybe it can be changed or it's a bug? Not sure why the ability wouldn't be used for if there's a single door available.

Reply:
yeah, I'd say we keep your change. The original looks like an oversight to me.